uPortal-soffit-renderer/src/main/java/org/apereo/portal/soffit/renderer/SoffitRendererController.java

```diff
@@ -249,6 +282,102 @@ private String getCompletePathforParts(final String... parts) {
 
     }
 
+    private Map<String,Object> gatherModelAttributes(String viewName, HttpServletRequest req, HttpServletResponse res,
+                        PortalRequest portalRequest, Bearer bearer, Preferences preferences, Definition definition) {
+
+        final Map<String,Object> rslt = new HashMap<>();
+
+        logger.debug("Processing model attributes for viewName='{}'", viewName);
+
+        for (Map.Entry<AnnotatedElement,Object> y : modelAttributes.entrySet()) {
```

Comment:
Would this be a place where [using Java 8 streams](https://docs.oracle.com/javase/8/docs/api/java/util/stream/package-summary.html) would be effective?